src/main/kotlin/simulator/io/Input.kt

```diff
@@ -0,0 +1,15 @@
+package simulator.io
+
+class Input {
+    fun getMoney(): Int {
+        println("구입금액을 입력해 주세요.")
+        return readln().toInt()
+    }
+
+    fun getLotto(): List<Int> {
+        println("지난 주 당첨 번호를 입력해 주세요.")
+        return readln().split(",")
+            .map { it.toInt() }
+            .toList()
```

Comment:
```suggestion
            .map { it.toInt() }
```
컬렉션과 관련된 함수들(filter, map 등)은 기본적으로 새로운 컬렉션을 만들어줍니다.
해서, map 함수를 호출하셨다면 toList 함수는 호출하지 않아도 됩니다. 😄